lib/local-storage.ts

```diff
@@ -0,0 +1,30 @@
+import { LocalUserChoices } from '@livekit/components-react';
+
+const LOCAL_STORAGE_KEY = 'livekit_meet';
+
+type LocalStorageData = {
+  userChoices: LocalUserChoices;
```

Comment:
For better or worse (arguably worse) the e2ee pw is part of the `LocalUserChoices`. I don't think we should store that in local storage. also the setting whether or not `e2ee` is enabled doesn't really belong there and would be different from meeting to meeting.

Reply:
yeah... we gotta get e2ee outta there..

Reply:
maybe hold this PR until that is done? otherwise it seems like things could get really confusing real fast.